packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -1,29 +1,51 @@
-import React, { ReactElement } from 'react';
+import React, { CSSProperties, ReactElement } from 'react';
+import classNames from 'classnames';
 import classed from '../../lib/classed';
 import { OnboardingHeadline } from './OnboardingHeadline';
 import AuthOptions, { AuthDisplay } from './AuthOptions';
 import { AuthTriggers } from '../../lib/auth';
 import { MemberAlready } from '../onboarding/MemberAlready';
 import { useAuthContext } from '../../contexts/AuthContext';
+import { useViewSize, ViewSize } from '../../hooks';
+import LoginButton from '../LoginButton';
 
-const Container = classed('div', 'flex flex-col w-[23.25rem]');
+const baseClasses =
+  'fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center';
+
+export const AuthBannerContainer = classed('div', baseClasses);
+
+const Section = classed('div', 'flex flex-col w-[23.25rem]');
+const style: CSSProperties = {
+  background: `linear-gradient(270deg, rgba(206, 61, 243, 0.16) 0%, rgba(113, 71, 237, 0.16) 100%), #0E1217`,
+};
 
 export function AuthenticationBanner(): ReactElement {
   const { showLogin } = useAuthContext();
+  const isLaptop = useViewSize(ViewSize.Laptop);
+
+  if (!isLaptop) {
+    return (
+      <LoginButton
+        style={style}
+        className={{
+          container: classNames(baseClasses, '!gap-2 px-4 py-2'),
```

Comment:
Yeah, we can probably use its existing gap, either way works for me.

Reply:
Ah yes something like you did for the typo fix in the other PR would work.
Just want to avoid overwrites where possible, rather accept default or pass them everywhere :)

Reply:
Ah, I see. It was just that 3 other places use this as well, so was not sure at first if it is going to be efficient, but, yeah, I think that is the right path.

Reply:
Done 🥳